tests/test_swc.py

```diff
@@ -14,19 +14,20 @@
 url = (top_level / "test_upload").as_uri()
 url_seg = url + "_segments"
 url = url + "/serial"
-if not (Path(url[5:]) / "info").is_file():
-    print("Uploading data.")
-    upload_volumes(input, url, 1)
-if not (Path(url_seg[5:]) / "info").is_file():
-    print("Uploading segmentataion.")
-    upload_segments(input, url_seg, 1)
-assert (Path(url[5:]) / "info").is_file()
-assert (Path(url_seg[5:]) / "info").is_file()
+# if not (Path(url[5:]) / "info").is_file():
+#     print("Uploading data.")
+#     upload_volumes(input, url, 1)
+# if not (Path(url_seg[5:]) / "info").is_file():
+#     print("Uploading segmentataion.")
+#     upload_segments(input, url_seg, 1)
+# assert (Path(url[5:]) / "info").is_file()
+# assert (Path(url_seg[5:]) / "info").is_file()
 
 URL = str(Path(__file__).resolve().parents[0] / "upload")
 # read in s3 path to dataframe
-s3_path = "s3://mouse-light-viz/precomputed_volumes/brain1_segments"
-df_s3 = swc.read_s3(s3_path, seg_id=2, mip=6)
+# s3_path = "s3://mouse-light-viz/precomputed_volumes/brain1_segments"
+s3_path = url_seg
```

Comment:
is there a reason you have two different variables, one called s3_path and the other called url_seg?